test/tools/WebListener/Program.cs

```diff
@@ -62,7 +62,10 @@ public static void Main(string[] args)
                         {
                             var certificate = new X509Certificate2(args[0], args[1]);
                             HttpsConnectionAdapterOptions httpsOption = new HttpsConnectionAdapterOptions();
-                            httpsOption.SslProtocols = SslProtocols.Tls11;
+
+                            // TLS 1.1 is obsolete. Using this value now defaults to TLS 1.2.
+                            httpsOption.SslProtocols = SslProtocols.Tls12;
+
```

Comment:
If we think the tests are still reasonable to keep, maybe we just suppress the warning?

Reply:
It causes a build failure if not suppressed. I am not sure if we should have an unsecure protocol in our test tool.